src/libsync/owncloudpropagator.cpp

```diff
@@ -691,15 +691,16 @@ bool OwncloudPropagator::createConflict(const SyncFileItemPtr &item,
     emit touchedFile(fn);
     emit touchedFile(conflictFilePath);
 
+    // If the file is locked, we want to retry this sync when it
+    // becomes available again.
+    if (FileSystem::isFileLocked(fn, FileSystem::LockMode::Exclusive)) {
+        emit seenLockedFile(fn, FileSystem::LockMode::Exclusive);
+        if (error)
+            *error = tr("File %1 is currently in use").arg(fn);
+        return false;
+    }
```

Comment:
before this new block, `touchedFile()` is emitted. However, if the file is locked, we do not touch anything, so the emits should probably be moved below.